frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/GHPRLoader.java

```diff
@@ -0,0 +1,40 @@
+package com.virtuslab.gitmachete.frontend.actions.common;
+
+import java.util.List;
+
+import com.intellij.openapi.project.Project;
+import lombok.val;
+import org.jetbrains.plugins.github.api.GHRepositoryCoordinates;
+import org.jetbrains.plugins.github.api.GithubServerPath;
+import org.jetbrains.plugins.github.api.data.pullrequest.GHPullRequestShort;
+import org.jetbrains.plugins.github.pullrequest.data.GHPRDataContextRepository;
+import org.jetbrains.plugins.github.util.GithubUrlUtil;
+
+import com.virtuslab.gitmachete.frontend.ui.providerservice.SelectedGitRepositoryProvider;
+
+public class GHPRLoader {
+
+  public static List<GHPullRequestShort> loadPRs(Project project) {
+    try {
+      val repository = GHPRDataContextRepository.Companion.getInstance(project);
```

Comment:
> 1. GH plugin must be initialised by a user (by opening a window provided by a plugin)

What specifically happens if it does NOT happen? what specific call fails and with what exception? 🤔

Reply:
Huh okay... and do we have any way to open GitHub tool window programmatically? annoying as it may be

Reply:
Rel: `OpenMacheteTabAction`